### ISSUE TITLE
Installing Plugins via Docs

### ISSUE THREAD

**Issue by trepersec** [opensearch-project/OpenSearch-Dashboards#366]
**Describe the bug**

Following the documentation listed on https://docs-beta.opensearch.org/docs/opensearch-dashboards/install/plugins/ . I am installing the plugins but the links aren't being referenced properly and causing an error.

**To Reproduce**
1. Go to https://docs-beta.opensearch.org/docs/opensearch-dashboards/install/plugins/
2. Under the Install subcategory, copy and paste one of the commands to install a plugin
3. Error

**Expected behavior**
Correct resolution of the link to the plugin

**OpenSearch Version**
1.0.0-beta1

**Dashboards Version**
Please list the version of OpenSearch Dashboards being used.

**Plugins**

Please list all plugins currently enabled.

**Screenshots**

**Host/Environment (please complete the following information):**
 - Browser and version [e.g. 22]

**Additional context**

**Comment by dblock** [comment]
That file, "https://d3g5vo6xdbdb9a.cloudfront.net/downloads/opensearch-dashboards-plugins/opensearch-alerting/opensearchAlertingOpenSearch Dashboards-1.0.0.0.zip" doesn't exist. I see two problems: 1) the file name has spaces and what not and should become kebab-case, 2) it wasn't published part of the beta

@VachaShah this is somewhat related to #357, could you please followup on plugins producing artifacts that match kebab case and not the plugin ID

**Comment by trepersec** [comment]
> That file, "https://d3g5vo6xdbdb9a.cloudfront.net/downloads/opensearch-dashboards-plugins/opensearch-alerting/opensearchAlertingOpenSearch Dashboards-1.0.0.0.zip" doesn't exist. I see two problems: 1) the file name has spaces and what not and should become kebab-case

I've tried different combinations of the link as I had the same assumptions, but none resolved:
e.g.
- /opensearchAlertingOpenSearchDashboards-1.0.0.0.zip
- /OpenSearchDashboards-1.0.0.0.zip
- /AlertingOpenSearchDashboards-1.0.0.0.zip

Obviously, I could just not know the correct combination, but still wanted to relay the issue. Thank you!

**Comment by dblock** [comment]
@trepersec no, it doesn't exist in any form AFAIK, but I could be wrong: @bbarani what's the plan for these?

**Comment by VachaShah** [comment]
> That file, "https://d3g5vo6xdbdb9a.cloudfront.net/downloads/opensearch-dashboards-plugins/opensearch-alerting/opensearchAlertingOpenSearch Dashboards-1.0.0.0.zip" doesn't exist. I see two problems: 1) the file name has spaces and what not and should become kebab-case, 2) it wasn't published part of the beta
> 
> @VachaShah this is somewhat related to #357, could you please followup on 1), ie. plugins should be producing artifacts that match kebab case and not the plugin ID

This PR has been merged now.

**Comment by dblock** [comment]
I've assigned this one to you @VachaShah and renamed.
